tests/Api/Responses/Expected/shop/product/get_product_variant_with_default_locale_translation.json

```diff
@@ -5,8 +5,11 @@
   "code": "MUG_BLUE",
   "name": "Blue Mug",
   "product": "\/api\/v2\/shop\/products\/MUG",
-  "optionValues": [
-    "\/api\/v2\/shop\/product-option-values\/COLOR_BLUE"
+  "options": [
+      {
+          "option": "\/api\/v2\/shop\/product-options\/COLOR",
+          "optionValue": "\/api\/v2\/shop\/product-option-values\/COLOR_BLUE"
+      }
```

Comment:
I have mixed feelings about this :/. In situation when we have 10 options we're going to make 20 requests to get all options with values. Am I right?

Reply:
@NoResponseMate wdyt?

Reply:
Actually, you are right. Previously we had to fetch every optionValue where we could fetch options, now we have all the IRIs in place, but you still have to perform multiple requests. I would be in favor of embedding the object where appropriate, like here, but in the Sylius philosophy we serialize less rather than more